### PR TITLE
(GH-15) Added .NET Core target

### PR DESCRIPTION
**Pull request by RLittlesII**
This pull request is to resolve issue #15.

@devlead I am seeing some netstandard issues with the LitJson library.  I migrated WebClient to HttpClient.  Thoughts on how I can resolve these issues are appreciated.

### CODE REVIEW

**Comment by devlead** [comment]
Think this is a WIPI that sorts the. NET standard issues
https://github.com/cake-contrib/Cake.MicrosoftTeams/tree/master/src/Cake.MicrosoftTeams/LitJson

File: src/Cake.Slack/Properties/Namespaces.cs [hunk 1]
```diff
@@ -1,23 +0,0 @@
-using System.Runtime.CompilerServices;
```
Comment (devlead) [inline src/Cake.Slack/Properties/Namespaces.cs]: Why is the name space documentation removed?

Comment (RLittlesII) [reply src/Cake.Slack/Properties/Namespaces.cs]: I had issues with property folders when porting to .NET Core on a few other projects.  I will add it back and verify that there are no issues.
More importantly, are there plans to source the LitJson using the T4 template or is this sufficient for now to target .NET Core?  When I attempted it, it kept the `Cake.MicrosoftTeams` in the namespace which was less than ideal.

Comment (devlead) [reply src/Cake.Slack/Properties/Namespaces.cs]: Doesn't have to be in properties,  can be wherever, but it it needs to be in the assembly for cake website to pick it up 😎

Don't know if T4 is needed, probably for now fine to correct namespace and ensure internal.

Comment (RLittlesII) [reply src/Cake.Slack/Properties/Namespaces.cs]: I didn't realized that was needed for the website 😳.  I'll get this cleaned up and tested sometime this weekend.

**Comment by RLittlesII** [comment]
@devlead Converted build to Cake.Recipe because it was just the easiest way to get it building.  I kicked the tires with the `slack.cake` that you have in the repo.  After this gets merged, I can wire up integration tests.  

Please verify the following:
- nuspec is in the order
- Cake.Slack API path works (I was able to test webhooks)

**Comment by devlead** [comment]
@RLittlesII cool, think AppVeyor failure is due to tools\package.config not committed or right cake version should be sub 0.22.0 for Cake.Recipie or skip verification supplied in bootstrapper.

**Comment by RLittlesII** [comment]
@devlead I still need to commit the `nuspec\nuget` folder so it will build packages.  I seem to be failing at .gitignore and adding all the changes.

File: src/Cake.Slack/Properties/Namespaces.cs (hunk 1 above)
Comment (RLittlesII) [reply src/Cake.Slack/Properties/Namespaces.cs]: @devlead This should be resolved with the latest rebase.

Comment (devlead) [reply src/Cake.Slack/Properties/Namespaces.cs]: Splendid, will ping back as soon as I have reviewed.

**Comment by gep13** [comment]
@RLittlesII @devlead what is the status of this PR?  Does it just need a final review?

**Comment by RLittlesII** [comment]
@gep13 There was a change requested.  I believe I satisfied it, but it doesn't appear as though it took in the review.

**Pull request merged**